scripts/generate-dev-environment.sh

```diff
@@ -22,6 +23,7 @@ PRODUCTS_URL=https://localhost:9005
 LEDGER_URL=https://localhost:9007
 PUBLIC_AUTH_BASE=http://localhost:9006
 PUBLIC_AUTH_URL=http://localhost:9006/v1/frontend/auth
+WEBHOOKS_URL=https://localhost:9008
```

Comment:
don't think we need this block anymore and this update as we can't tunnel to our environments